_download-meta/catcodec-releases.md

```diff
@@ -1,3 +1,3 @@
 ---
-folder: extra/catcodec
+folder_old_infrastructure: extra/catcodec
```

Comment:
When you get around to making a readme, I'd recommend a list of all the "custom" variables (and what they're for)

Reply:
These are temporary during migration, but I agree. Will do.